examples/complex_wallet_examples/create_faucet_server/internal/config/config.go

```diff
@@ -0,0 +1,85 @@
+package config
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/bsv-blockchain/go-wallet-toolbox/pkg/defs"
+)
+
+type Config struct {
+	Port int `mapstructure:"port"`
+
+	// Faucet configuration
+	ServerPrivateKey string          `mapstructure:"server_private_key"`
+	FaucetPrivateKey string          `mapstructure:"faucet_private_key"`
```

Comment:
What is the difference between those keys?
One key should be enough, we need only private key for faucet.